conformance/lang/expressions/regexp/regexp_semantics.balt

```diff
@@ -0,0 +1,719 @@
+Test-Case: output
+Description: Test basic regular expression.
+Labels: RegExp, string
+
+import ballerina/lang.regexp;
+
+function init() {
+    string:RegExp reg1 = re `([A-D]+|[H-Z]*)|C{2,9}`;
+    io:println(reg1); // @output ([A-D]+|[H-Z]*)|C{2,9}
+
+    regexp:RegExp reg2 = re `([A-D]+|[H-Z]*)|C{2,9}`;
+    io:println(reg2); // @output ([A-D]+|[H-Z]*)|C{2,9}
+}
+
+Test-Case: output
+Description: Test basic regular expression at module-level.
+Labels: RegExp, module-var-decl, string
```

Comment:
we should use `module-init-var-decl` or `module-no-init-var-decl`. Let's fix wherever applicable